lib/usd/ui/views/TreeItem.cpp

```diff
@@ -41,7 +50,36 @@ int TreeItem::type() const
 	return ParentClass::ItemType::UserType;
 }
 
-void TreeItem::setCheckState(Qt::CheckState st)
+const QPixmap& TreeItem::checkImage() const
+{
+	if (fsCheckBoxOn == nullptr)
+	{
+		const TreeModel* treeModel = qobject_cast<const TreeModel*>(model());
+		if (treeModel != nullptr)
+		{
```

Comment:
Feels like maybe there should be at least a `TF_VERIFY` (or its equivalent) here? If for whatever reason `treeModel` is invalid here, it looks like we could end up derefencing `nullptr` below?